### PR TITLE
🎓 Origin topic --- Invisible hand 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Adding the section that explains how there is no point to anything. 

### Additional Notes

This one is kinda' tricky as I don't think people can truly appreciate these ideas unless they spend some time thinking about them on their own. Either way, I think this will hopefully hit the point.

### CODE REVIEW

File: site/topics/origin/origin.rst [hunk 1]
```diff
@@ -209,6 +209,40 @@ Turing Completeness in Simple Systems
...
+
+* Virtually nothing within the universe exists for a purpose
+
+    * The exceptions being things like human created things
+
+
+* Things only exist because they don't not
+* Evolution has no intelligence guiding it
+* Yet, things seem to *work*
+* But remember, they works, because if it did not, it would exist
```
Comment (twentylemon) [inline site/topics/origin/origin.rst:225]: english exists, but not here

Comment (twentylemon) [reply site/topics/origin/origin.rst:225]: selecting a single line is hard. line 225 doesn't read well

Comment (jameshughes89) [reply site/topics/origin/origin.rst:225]: they works

Comment (twentylemon) [reply site/topics/origin/origin.rst:225]: image

**Pull request merged**